### PR TITLE
Release/1.0.0 rc1

### PR DESCRIPTION
**Pull request by croomes**
Initial draft

### CODE REVIEW

File: _docs/reference/release_notes.md [hunk 1]
```diff
@@ -5,43 +5,126 @@ anchor: reference
...
+  information and log files.  This is currently only available via the Web
+  interfce or the API.
+- Kubernetes mount option support (requires Kubernetes 1.10).
+
+### Improved
+
+- Performance has improved significantly throughout and has been thoroughly
+  tested on a variety of workloads.  Specific areas include optimisations for
+  larger block sizes and improvements to the caching engine.
+- Volumes will go read-only shortly before the underlying device runs out of of
```
Comment (darkowlzz) [inline _docs/reference/release_notes.md]: an extra "of"

**Comment by croomes** [comment]
@darkowlzz updated - well spotted!

File: _docs/reference/release_notes.md [hunk 2]
```diff
@@ -5,43 +5,126 @@ anchor: reference
...
+- Several internal data structures were changed and are not compatible with
+  previous versions.  For this reason upgrades from version 0.10.x and earlier
+  are not supported.
+- The API endpoint `/v1/metrics` has been replaced with `/metrics` to conform
+  with Prometheus best practices.
+
+### New
+
+- Volume presentation has changed to use the SCSI subsystem via the
+  Linux SCSI Target driver. Previous versions used NBD where available, or FUSE
```
Comment (Arau) [inline _docs/reference/release_notes.md]: I would add a reference to the compatibility page for LIO reference/os_support

Comment (Arau) [reply _docs/reference/release_notes.md]: other than that I find the docs very precise and well explained.

Comment (oicheryl) [reply _docs/reference/release_notes.md]: @Arau 

_docs/reference/os_support needs to be added to _guides/006-reference.md to show up in the left hand menu.

Comment (Arau) [reply _docs/reference/release_notes.md]: @oicheryl  that was added on the lio_support branch as it will merge to this one I believe.

**Pull request merged**